src/views/Instance/Instance.js

```diff
@@ -101,6 +101,42 @@ const Instance = (props) => {
       noData: t('commoncomponents.cytoviz.elementData.noData', 'No data to display for this element.'),
       attributes: {},
     },
+    accordeon: {
```

Comment:
accordIon :laughing: and it needs to be corrected in translation's path :slightly_smiling_face: